glustercli/cmd/volume.go

```diff
@@ -302,6 +302,7 @@ func volumeInfoDisplay(vol api.VolumeGetResp) {
 	fmt.Println("Type:", vol.Type)
 	fmt.Println("Volume ID:", vol.ID)
 	fmt.Println("State:", vol.State)
+	fmt.Print("Capactiy: ", vol.Capacity, "MB\n")
```

Comment:
Showing in MB is not be user friendly in CLI, we can change this to `humanReadable(vol.Capacity)`. That function should take input in MB and returns string like `1TB` or `10GB` etc

Reply:
done

Reply:
@harigowtham i don't see a function as @aravindavk suggested

Reply:
@Madhu-1 I have wrote one in the utils file.